<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sms</class>
 <widget class="QDialog" name="Sms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Sms{
background-image:url(:/Resources/client_img/bg.png);
}</string>
  </property>
  <widget class="QLineEdit" name="numSms">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>271</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#numSms{
background-color: #F9FDFF;
border:1px solid rgba(222, 226, 230, 0.8);
padding-left:5px;
border-radius:10px;
color:#585856
}
#numSms:focus{
border:1px solid #6568f3;
outline:none;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>111</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Destinataire</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="msgSms">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>271</width>
     <height>86</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#msgSms{
background-color: #F9FDFF;
border:1px solid rgba(222, 226, 230, 0.8);
padding-left:5px;
border-radius:10px;
color:#585856
}
#msgSms:focus{
border:1px solid #6568f3;
outline:none;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>101</width>
     <height>86</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendSmsBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>51</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 	background-color: rgb(255, 114, 68);
	border:none;
	border-radius:15px;
}

QPushButton::hover {
  	background-color: rgb(254, 129, 75);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Client_resources.qrc">
     <normaloff>:/Resources/client_img/send_icon.png</normaloff>:/Resources/client_img/send_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>17</width>
     <height>17</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>81</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Num√©ro</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="destSms">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>271</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#destSms{
background-color: #F9FDFF;
border:1px solid rgba(222, 226, 230, 0.8);
padding-left:5px;
border-radius:10px;
color:#585856
ReadOnly(true);
}
#destSms:focus{
border:1px solid #6568f3;
outline:none;
}
</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Client_resources.qrc"/>
 </resources>
 <connections/>
</ui>
